src/network/networkctl.c

```diff
@@ -92,19 +93,30 @@ JsonFormatFlags arg_json_format_flags = JSON_FORMAT_OFF;
 
 STATIC_DESTRUCTOR_REGISTER(arg_drop_in, freep);
 
-static int check_netns_match(sd_bus *bus) {
-        _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+static int check_netns_match(void) {
         struct stat st;
         uint64_t id;
+        JsonVariant *reply = NULL;
+        _cleanup_(varlink_unrefp) Varlink *vl = NULL;
         int r;
 
-        assert(bus);
+        static const JsonDispatch dispatch_table[] = {
```

Comment:
If you move this to the beginning of the function, then please move it to the very beginning, then declare non-static variables.
```
static const JsonDispatch dispatch_table[] = {
        ...,
};

_cleanup_(varlink_unrefp) Varlink *vl = NULL;
JsonVariant *reply = NULL;
struct stat st;
...
```
(But it is also OK to locate the table just before `json_variant_dispatch()`, link we do in resolvectl.c. There is no strict coding style rule about that.)

Reply:
Ack, I think resolvectl.c was my point of reference. Will move it back there!